examples/states_ui/util.rs

```diff
@@ -0,0 +1,25 @@
+use amethyst::{
+    core::transform::ParentHierarchy,
+    ecs::{
+        error::WrongGeneration,
+        prelude::{Entity, World, WorldExt},
+    },
+};
+
+use std::iter;
+
+/// delete the specified root entity and all of its descendents as specified
+/// by the Parent component and maintained by the ParentHierarchy resource
+// from https://github.com/amethyst/evoli src/utils/hierarchy_util.rs
```

Comment:
Missed a slash

Reply:
that is actually intentional, as it is no link

Reply:
ah, initially I meant the first slash, then saw there's a middle slash that isn't there, but I'm guessing both are intentional